options.go

```diff
@@ -57,6 +57,18 @@ func IgnoreTopFunction(f string) Option {
 	})
 }
 
+// IgnoreCurrentStacks remembers all current stacks and ignores them on verifying.
+// This Option may be used in big projects that recently started to use go-leak.
```

Comment:
Let's make this a little more verbose, and remove references to stack, since it actually ignores goroutines rather than the stack

```suggestion
// IgnoreCurrent records all current goroutines when the option is created, and ignores
// them in any future Find/Verify calls.
```

It may be useful to explain / show in an example that this can be used with `defer` as normal too (since the option is evaluated when the line is executed, while the verification is deferred)

Reply:
Thanks, fixed